newswires/app/db/FingerpostWireEntry.scala

```diff
@@ -109,14 +113,13 @@ object FingerpostWireEntry extends SQLSyntaxSupport[FingerpostWireEntry] {
                         | $whereClause
                         | ORDER BY ${FingerpostWireEntry.syn.ingestedAt} DESC
                         | LIMIT $effectivePageSize
-                        | OFFSET $position
                         | """.stripMargin
       .map(FingerpostWireEntry(syn.resultName))
       .list()
       .apply()
 
     val keywordCounts = getKeywords(additionalWhereClauses =
-      whereClauses
+      commonWhereClauses
```

Comment:
should we be applying the 'beforeId' clause to the keyword counts query as well?

I can see an argument either way..

Reply:
no I don't think so, I feel keyword counts ought to always be latest counts given the search terms (excluding `beforeId` and `sinceId`) I can picture this being the easiest on the client